developers-italia-api/custom.yaml

```diff
@@ -1,3 +1,8 @@
+deploymentAnnotations:
+  keel.sh/policy: major
+  keel.sh/trigger: poll      
+  keel.sh/pollSchedule: "@every 1h"
```

Comment:
We use `1m` [with the onboarding app](https://github.com/teamdigitale/devita-infra-kubernetes/blob/20579a77bb0dd65372039b6c92cd1a3e3c9225d1/onboarding/templates/deployment.yaml#L19)

I'm afraid that 1 hour would result in assuming the deploy mechanism is broken when a tag is made and the deploy is not done several minutes later.

Is there something that doesn't allow us to lower it?